docs/resources/schema.md

```diff
@@ -33,28 +57,277 @@ resource "snowflake_schema" "schema" {
 
 ### Optional
 
+- `catalog` (String) The database parameter that specifies the default catalog to use for Iceberg tables.
 - `comment` (String) Specifies a comment for the schema.
-- `data_retention_days` (Number) Specifies the number of days for which Time Travel actions (CLONE and UNDROP) can be performed on the schema, as well as specifying the default Time Travel retention time for all tables created in the schema. Default value for this field is set to -1, which is a fallback to use Snowflake default.
-- `is_managed` (Boolean) Specifies a managed schema. Managed access schemas centralize privilege management with the schema owner.
-- `is_transient` (Boolean) Specifies a schema as transient. Transient schemas do not have a Fail-safe period so they do not incur additional storage costs once they leave Time Travel; however, this means they are also not protected by Fail-safe in the event of a data loss.
-- `tag` (Block List, Deprecated) Definitions of a tag to associate with the resource. (see [below for nested schema](#nestedblock--tag))
+- `data_retention_time_in_days` (Number) Specifies the number of days for which Time Travel actions (CLONE and UNDROP) can be performed on the database, as well as specifying the default Time Travel retention time for all schemas created in the database. For more details, see [Understanding & Using Time Travel](https://docs.snowflake.com/en/user-guide/data-time-travel).
+- `default_ddl_collation` (String) Specifies a default collation specification for all schemas and tables added to the database. It can be overridden on schema or table level. For more information, see [collation specification](https://docs.snowflake.com/en/sql-reference/collation#label-collation-specification).
+- `enable_console_output` (Boolean) If true, enables stdout/stderr fast path logging for anonymous stored procedures.
+- `external_volume` (String) The database parameter that specifies the default external volume to use for Iceberg tables.
+- `is_transient` (Boolean) Specifies the schema as transient. Transient schemas do not have a Fail-safe period so they do not incur additional storage costs once they leave Time Travel; however, this means they are also not protected by Fail-safe in the event of a data loss.
```

Comment:
shouldn't all the boolean attributes (NOT sf params) be our boolean strings?

Reply:
Why? In Snowflake, this is not a boolean field, but an option during create.

Reply:
Because currently the proper behavior of this field assumes that SF default is also fault but it will fail with the default change. It won't with the boolstring.

Also, consistency in the provider.

Reply:
Changed to BooleanString (except the new bool parameter, but in databases there are also bool parameters).